.github/workflows/zombienet_substrate.yml

```
@@ -1,45 +1,140 @@
 name: Zombienet Substrate
 
 on:
-  workflow_run:
-    workflows: [Build and push images]
-    types: [completed]
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, labeled]
```

```suggestion
    types: [opened, synchronize, reopened, ready_for_review]
```
I'd remove `labeled` since it shouldn't be re-triggered on new labels